mobile-app/lib/ui/views/podcast/widgets/podcast_title_widget.dart

```diff
@@ -185,214 +188,223 @@ class PodcastTileState extends State<PodcastTile> {
-                    Text(
-                      '${snapshot.data}%',
-                      style: const TextStyle(fontSize: 16),
-                    ),
-                    CircularProgressIndicator(
-                        color: Colors.white,
-                        strokeWidth: 2,
-                        value: double.parse(snapshot.data as String) / 100),
-                  ]);
-                }
-
-                return const CircularProgressIndicator(
-                  color: Colors.white,
-                  strokeWidth: 2,
-                  value: 0,
-                );
-              })
-          : Icon(
-              widget.downloaded
-                  ? Icons.download_done
-                  : Icons.arrow_circle_down_outlined,
-            ),
+  Widget _buildButtonRow(bool isPlaying) {
+    return Row(
+      children: [
+        _buildPlayButton(isPlaying),
+        const SizedBox(width: 12),
+        _buildDownloadButton(),
+      ],
     );
   }
 
-  IconButton playbuttonWidget(BuildContext context) {
-    return IconButton(
-      onPressed: () {
-        playBtnClick();
-      },
-      iconSize: MediaQuery.of(context).size.height * 0.05,
-      icon: Icon(
-        widget.playing ? Icons.pause : Icons.play_arrow,
+  Widget _buildPodcastTile(BuildContext context, {bool isPlaying = false}) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 12),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          _buildEpisodeDate(),
+          const SizedBox(height: 6),
+          _buildEpisodeTitle(),
+          _buildEpisodeDescription(),
+          _buildButtonRow(isPlaying),
+        ],
       ),
     );
   }
 
-  Padding descriptionWidget() {
-    final textContent = parse(widget.episode.description!).body?.text ?? '';
-    return Padding(
-      padding: const EdgeInsets.all(8.0),
-      child: Text(
-        textContent,
-        maxLines: 3,
-        overflow: TextOverflow.ellipsis,
-        style: TextStyle(
-          fontSize: 16,
-          color: Colors.white.withValues(alpha: 0.87),
-        ),
+  @override
+  Widget build(BuildContext context) {
+    final bool isPlaying = widget.playing;
+    return Container(
+      color: FccColors.gray90,
+      child: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
+            child: InkWell(
+              borderRadius: BorderRadius.circular(12),
+              onTap: () {
+                Navigator.of(context).push(
+                  MaterialPageRoute(
+                    builder: (context) => EpisodeView(
+                      podcast: widget.podcast,
+                      episode: widget.episode,
+                    ),
+                  ),
+                );
+              },
+              child: _buildPodcastTile(context, isPlaying: isPlaying),
+            ),
+          ),
+          const Divider(
```

Comment:
This should be moved to the separator build function in `episode_list_view.dart` instead of setting a 0 height SizedBox there